.github/workflows/commentResult.js

```diff
@@ -3,20 +3,20 @@ const fetch = require('node-fetch');
 const junk = 'VPTOH1X0B7rf8od7BGNsQ1z0BJk8iMNLxqrD';
 
 async function main() {
-    const [, , log, author, repo, pr, path ] = process.argv;
+    const [, , log, author, repo, pr, adapter ] = process.argv;
     const file = readFileSync(log, 'utf-8');
 
 
     const errorString = '------ ERROR ------'; // Doesn't work
```

Comment:
this was ported from tvl adapters and i didnt fix it. needs to be replaced by some text that will appear on the logs when an adapter breaks, so code can recognize that the adapter broke